gammapy/background/models.py

```diff
@@ -71,3 +82,724 @@ def evaluate(self, x, y):
+            self.to_fits_image().writeto(outfile, **write_kwargs)
+        else:
+            raise ValueError("Invalid format {}.".format(format))
+
+    @property
+    def image_extent(self):
+        """Image extent `(x_lo, x_hi, y_lo, y_hi)`.
+
+        Returns
+        -------
+        im_extent : `~astropy.coordinates.Angle`
+            array of bins with the image extent
+        """
+        bx = self.detx_bins
+        by = self.dety_bins
+        return Angle([bx[0], bx[-1], by[0], by[-1]])
+
+    @property
+    def spectrum_extent(self):
+        """Spectrum extent `(e_lo, e_hi)`.
+
+        Returns
+        -------
+        spec_extent : `~astropy.units.Quantity`
+            array of bins with the spectrum extent
+        """
+        b = self.energy_bins
+        return Quantity([b[0], b[-1]])
+
+    @property
+    def image_bin_centers(self):
+        """Image bin centers `(x, y)`.
+
+        Returns
+        -------
+        det_edges_centers : `~astropy.coordinates.Angle`
+            array of bins with the image bin centers [[x_centers], [y_centers]]
+        """
+        detx_edges_low = self.detx_bins[:-1]
+        detx_edges_high = self.detx_bins[1:]
+        detx_edges_centers = (detx_edges_low + detx_edges_high)/2.
+        dety_edges_low = self.dety_bins[:-1]
+        dety_edges_high = self.dety_bins[1:]
+        dety_edges_centers = (dety_edges_low + dety_edges_high)/2.
+        return Angle([detx_edges_centers, dety_edges_centers])
+
+    @property
+    def energy_bin_centers(self):
+        """Energy bin centers (logarithmic center).
+
+        Returns
+        -------
+        energy_bin_centers : `~astropy.units.Quantity`
+            array of bins with the spectrum bin centers
+        """
+        energy_edges_low = self.energy_bins[:-1]
+        energy_edges_high = self.energy_bins[1:]
+        e_lo_tev = energy_edges_low.to('TeV').value
+        e_hi_tev = energy_edges_high.to('TeV').value
+        energy_bin_centers = 10.**(0.5*(np.log10(e_lo_tev*e_hi_tev)))
+        # TODO: this function should be reviewed/re-written, when
+        # the following PR is completed:
+        # https://github.com/gammapy/gammapy/pull/290
+        # as suggested in:
+        # https://github.com/gammapy/gammapy/pull/292#discussion_r34412865
+        return Quantity(energy_bin_centers, 'TeV')
+
+    @property
+    def det_wcs(self):
+        """WCS object describing the coordinates of the det (X, Y) bins.
+
+        This method gives the correct answer only for linear X, Y binning.
+
+        Returns
+        -------
+        wcs : `~astropy.wcs.WCS`
+            WCS object describing the bin coordinates
+        """
+        wcs = linear_wcs_from_arrays(name_x="DETX",
+                                     name_y="DETY",
+                                     bin_edges_x=self.detx_bins,
+                                     bin_edges_y=self.detx_bins)
+        return wcs
+
+    def find_det_bin(self, det):
```

Comment:
Done. I didn't add a `get_det_bin_value` yet, since it's not needed at the moment.